OSSIA/ossia-qt/device/qml_logger.cpp

```diff
@@ -187,10 +185,10 @@ static void LogQtToOssia(
   auto basename_arr = QFileInfo(context.file).baseName().toUtf8();
   auto filename = basename_arr.constData();
 
- if(m_globalLogfilter.contains(msg))
- {
-     emit qml_logger::instance().filteredLog(type,filename,context.line,msg);//type,context,msg);
-     return;
+  if(qml_logger::instance().logFilter().contains(msg))
+  {
+    emit qml_logger::instance().filteredLog(type,filename,context.line,msg);
```

Comment:
please don't repeat the call to qml_logger::instance().

instead do 

    auto& logger = qml_logger::instance();
    if(logger.logFilter().contains(...)) { 
      emit logger.filteredLog(...);
    }